src/components/message/message.stories.mdx

```diff
@@ -0,0 +1,38 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import messageDemo from './demo/default-message.twig';
+import messageWithCloseDemo from './demo/message-with-close.twig';
+import './message.scss';
+
+<Meta title="Components/Message" decorators={[withKnobs]} />
+
+# Message
+
+Use to convey general information or messages that suggest action for users.
+Message content should focus on a single theme, piece of information, or
+required action to avoid overwhelming users.
+
+Additionally, the message pattern should be used thoughtfully and sparingly for
+only the most important information.
```

Comment:
I think we can condense, and we should include the inspiration for this component so its purpose is not hypothetical. Example:

> Containing element for a short, actionable chunk of text. Originally designed to notify the user when they've lost their network connection.

I'm not sure that's exactly right, but hopefully you get the idea.